src/internal/connector/exchange/attachment.go

```diff
@@ -55,19 +60,27 @@ func uploadAttachment(
 
 	// item Attachments to be skipped until the completion of Issue #2353
 	if attachmentType == models.ITEM_ATTACHMENTTYPE {
-		name := ""
-		if attachment.GetName() != nil {
-			name = *attachment.GetName()
+		prev := attachment
+		attachment, err = support.ToItemAttachment(attachment)
+
```

Comment:
Nit: standard should be to cuddle funcs that produce errors with `if err != nil` conditionals.
```suggestion

		attachment, err = support.ToItemAttachment(attachment)
```

Reply:
Resolved within [ba6118d](https://github.com/alcionai/corso/pull/2355/commits/ba6118d34c7bf97dda9dea0bd92b9663b1f9820b)